packages/@okta/i18n/src/properties/login.properties

```diff
@@ -827,6 +827,14 @@ oie.on_prem.enroll.username.label = Enter authenticator username
 oie.on_prem.enroll.passcode.label = Enter authenticator passcode
 oie.on_prem.verify.title = Verify with {0}
 oie.on_prem.verify.passcode.label = Enter code
+idx.error.on-prem.token.change = Wait for token to change, then enter the new tokencode
```

Comment:
can this key start also with `oie.on_prem.*`? Maybe `oie.on_prem.error.token.change`?

Reply:
these keys comes from server, if they can update that will be amazing and not just all future error keys.

Reply:
@valberbenetz-okta can we make this change? If you already added the error key , i can file a follow up JIRA to change it

Reply:
Yup no problem. I haven't opened my PR yet with these error codes.

Reply:
Thank you! will update the error key to oie.on_prem.error.token.change